content/news/playtest-20150524.md

```diff
@@ -0,0 +1,55 @@
+It has been a month of busy development. Time to get back to testing.
+
+The mayor changes this time around include:
+
+  - A whole new in-game map editor with all the benefits that come from it.
+  - Improved system integration with `openra://` server join URLs and instant `.orarep` playback.
+  - The Dune 2000 mod got building upgrades which re-creates the tech tree of the original.
+  - Several enhanced classic missions have been added to the Tiberian Dawn mod.
+  - In the Red Alert mod you will now be notified when enemy units appear in your sight.
+
+We discovered a packaging problem in our last stable Windows build. This playtest should increase performance up to 30% on Windows.
```

Comment:
How about something like "Unfortunately, an oversight in the way that we package our windows installer meant that the performance of our Windows release-20150429 builds was much worse than it should have been.  We have fixed this for playtest-XXX, so you should find the game runs about 30% faster!"?